mlir/test/Dialect/MIOpen/lowering_input_tensor_non_zero_padding.mlir

```diff
@@ -0,0 +1,79 @@
+// This tests checks the following aspects of lowering component:
+// * Input has three transformations in total
+// * Input has correct output_layout across transformations
```

Comment:
Should skip those two because it is already tested at `lowering_input_tensor_cyxk_cnhw_hnhw.mlir`. This applies to the below test cases.

Reply:
Will change the unit test.